docs/broker/README.md

```diff
@@ -51,6 +51,28 @@ spec:
 
 ## Usage
 
+### ClusterChannelProvisioner
+
+`Broker`'s use their `spec.channelTemplate` to create their internal `Channel`s, which dictate the durability guarantees of events sent to that `Broker`. If `spec.channemTemplate` is not specified, then the [default provisioner](https://github.com/knative/docs/blob/master/docs/eventing/channels/default-channels.md#setting-the-default-channel-configuration) for their namespace is used.
+
+#### Setup
+
+Have a `ClusterChannelProvisioner` installed and set as the [default provisioner](https://github.com/knative/docs/blob/master/docs/eventing/channels/default-channels.md#setting-the-default-channel-configuration) for the namespace you are interested in. For development, the [`in-memory` `ClusterChannelProvisioner`](https://github.com/knative/eventing/tree/master/config/provisioners/in-memory-channel#deployment-steps) is normally used.
```

Comment:
Again, change links to knative.dev

https://www.knative.dev/docs/eventing/channels/default-channels/

The second will need to be a github link still.

Reply:
Done.